src/work-allocation-2/components/find-person/find-person.component.html

```diff
@@ -0,0 +1,17 @@
+<h1 class="govuk-heading-l">
+  <span class="govuk-caption-l">{{title}}</span>
+  Find the person
+</h1>
+<div class="govuk-form-group">
+  <fieldset class="govuk-fieldset" aria-describedby="">
+    <div id="waste-hint" class="govuk-hint">
+      Type the name of the person and select them.
+    </div>
+    <input type="text" aria-label="Number" matInput [formControl]="findPersonControl" [matAutocomplete]="auto" class="govuk-input govuk-!-width-one-third">
+    <mat-autocomplete autoActiveFirstOption #auto="matAutocomplete">
+      <mat-option *ngFor="let option of filteredOptions | async" [value]="option.email" (onSelectionChange)="onSelectionChange(option)">
+        {{option.name}}
+      </mat-option>
+    </mat-autocomplete>
+  </fieldset>
+</div>
```

Comment:
Would this new component findPerson better to override the one in exisiting task-assignment-container.component.html?

Reply:
Mine is a generic component that can be used anywhere. Thanks